src/index.js

```diff
@@ -244,11 +244,17 @@ function createLoadableComponent(loadFn, options) {
 
     render() {
       if (this.state.loading || this.state.error) {
+        // NOTE(kevinb): webapp has an ErrorBoundary which will catch and log any
+        // exceptions.  By throwing the error instead of passing it to the `loading`
+        // component.  This ensures that the error will be bubbled up without devs
```

Comment:
Perhaps is just me tired.  But Im having a lil hard time understand 

```
NOTE(kevinb): webapp has an ErrorBoundary which will catch and log any exceptions.
By throwing the error instead of passing it to the `loading` component.
```

Reply:
I missed some words.  I'll fix that pronto.  I'm the one that's tired. 😓

Reply:
That should be better now.